lib/teiserver/coordinator/coordinator_lib.ex

```diff
@@ -249,6 +251,7 @@ Multiple locks can be engaged at the same time
       :everybody -> true
       :host -> CacheUser.is_moderator?(user) or host
       :moderator -> CacheUser.is_moderator?(user)
+      :admin -> CacheUser.has_any_role?(user, "Admin")
```

Comment:
nitpick: for consistency perhaps have a `is_admin?` function and use that.

Reply:
Done